x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -404,6 +407,32 @@ describe('when on the hosts page', () => {
       ).not.toBeNull();
     });
 
+    it('should include the link to reassignment in Ingest', async () => {
+      coreStart.application.getUrlForApp.mockReturnValue('/app/ingestManager');
+      const renderResult = render();
+      const linkToReassign = await renderResult.findByTestId('hostDetailsLinkToIngest');
+      expect(linkToReassign).not.toBeNull();
+      expect(linkToReassign.textContent).toEqual('Reassign Policy');
+      expect(linkToReassign.getAttribute('href')).toEqual(
+        `/app/ingestManager#/fleet/agents/${agentId}/activity?openReassignFlyout=true`
+      );
+    });
+
+    describe('when link to reassignment in Ingest is clicked', () => {
+      beforeEach(async () => {
+        coreStart.application.getUrlForApp.mockReturnValue('/app/ingestManager');
+        const renderResult = render();
+        const linkToReassign = await renderResult.findByTestId('hostDetailsLinkToIngest');
+        reactTestingLibrary.act(() => {
+          reactTestingLibrary.fireEvent.click(linkToReassign);
+        });
+      });
+
+      it('should navigate to logs without full page refresh', () => {
```

Comment:
wait can you take out `logs` here in the title too 🙏

Reply:
oops, done